src/manage/dashboard/index.md

```diff
@@ -1,71 +1,110 @@
+Use **Dashboard** in the left panel to go back to the homepage
 
 ## Projects
 **Projects** tab in the left panel displays the list overview of the projects that are available to you in this workspace.
-![Mergin Maps web dashboard project tab](./web-projects-tab.jpg "Mergin Maps web dashboard project tab")
+![Mergin Maps dashboard projects tab](./mergin-maps-dashboard-projects.jpg "Mergin Maps dashboard projects tab")
 
-- click on **+ Create** to [create a new project](../create-project/#create-a-project-through-merginmaps-com), 
-- use **Find Public Projects** to browse public projects
-- select a project to see its details
+Here, you can:
+- use the **Create project** button to [create a new project](../create-project/#create-a-project-through-merginmaps-com)
+- search project by name
+- **Browse community projects** to explore public projects
+- click on a project to see its details
 
-When you open a project, there is an option to [share it](../project-advanced/), download it and [clone it](../create-project/#clone-an-existing-project-through-merginmaps-com).
-![Mergin Maps web project details](./mergin-project.jpg "Mergin Maps web project details")
+When you click on a project, you will see the overview of its content. Files can simply be **uploaded** to the project using drag and drop. 
 
-- in **FILES** you will find all files associated with the project. Click on a file to see the details, download it or delete it.
-- [MAP (beta)](#web-map) contains the map overview of the project. Available only in *Team* plan.
-- **HISTORY** contains available [project versions](../project-details/). Here you can see what was changed by which user and download a previous version of the project to your computer.
-- in **SETTINGS** you can manage [access](../project-advanced/) to the project 
+In the right upper corner, you will find buttons to **download** and [**clone**](../create-project/#clone-an-existing-project-through-merginmaps-com) the project.
+![Mergin Maps dashboard project details](./mergin-maps-project-details.jpg "Mergin Maps dashboard project details")
 
-### Web Map
-**Web Map** feature is available for [Team subscription](https://merginmaps.com/pricing). It displays your <MainPlatformName /> project directly in <AppDomainNameLink />.
+### Project files
+In **Files**, you can see the list of all files associated with the project. 
 
-The content is updated when the project is synchronised. If you see the message "Map config does not exist, please try to update the project", sync the project and the web map should be generated.
+Click on a file to see when it was modified, its size, to download or delete it.
+![Detail of a file in Mergin Maps project](./mergin-maps-dashboard-file-detail.jpg "Detail of a file in Mergin Maps project")
+
+### Webmap
+**Webmap** feature is available for [Team subscription](https://merginmaps.com/pricing). 
```

Comment:
Edit text:
The **Webmap** feature is available for `[Team subscription](https://merginmaps.com/pricing).`